docs/Lexicon.md

```diff
@@ -162,6 +162,25 @@ written "dupe". Pronounced the same way as the first syllable of
 A value whose type is a protocol composition (including a single protocol
 and *zero* protocols; the latter is the `Any` type).
 
+## explicit module build
+
+A module build where all dependency modules are passed to the compiler
```

Comment:
`modules` here include both Swift and Clang modules (.swiftmodule and .pcm files more specifically).

Reply:
Good point, I'll add this clarification.